app/javascript/mastodon/features/status/index.js

```diff
@@ -46,7 +46,6 @@ const makeMapStateToProps = () => {
     me: state.getIn(['meta', 'me']),
     boostModal: state.getIn(['meta', 'boost_modal']),
     deleteModal: state.getIn(['meta', 'delete_modal']),
-    autoPlayGif: state.getIn(['meta', 'auto_play_gif']),
```

Comment:
Maybe we just need to replicate the `autoPlayGif` removal in the `glitch` Status component?

Reply:
We definitely will (and `me`, `boost_modal`, etc. once upstream changes all that as well) although I'm a little surprised that that would have this kind of an effect. Settings aren't stored in anything related to `'meta'` (they're in `'settings'` and `'local_settings'`)—the worst that I would *expect* this would do would make `autoPlayGif` always return `false`.

That said, who knows :P